internal/cli/cmd/clustermesh.go

```diff
@@ -418,6 +418,28 @@ func newCmdClusterMeshConnectWithHelm() *cobra.Command {
 	return cmd
 }
 
+func newCmdClusterMeshDisconnectWithHelm() *cobra.Command {
+	var params = clustermesh.Parameters{
+		Writer: os.Stdout,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "disconnect",
+		Short: "Disconnect from a remote cluster",
+		Run: func(cmd *cobra.Command, args []string) {
+			params.Namespace = namespace
+			cm := clustermesh.NewK8sClusterMesh(k8sClient, params)
+			if err := cm.DisconnectWithHelm(context.Background()); err != nil {
+				fatalf("Unable to disconnect clusters: %s", err)
+			}
+		},
+	}
+
+	addCommonConnectFlags(cmd, &params)
```

Comment:
Should be changed to the only flag which is registered in the disconnect case.

Reply:
Not sure I follow... But I think these are used by both connect and disconnect via getAccessForConnect

Reply:
Oh, that's right. The weird thing though is that they don't make much sense for disconnect as they are actually never used (for instance the remote endpoint).

Reply:
OK - Good point indeed! Tx for the clarification! 
Took another swing at it. Let's see if we're happier...